src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
```

Comment:
do not apply any rules to the `*` selector except `box-sizing`

Reply:
not fixed

Reply:
Please remove margin: 0 from this selector. It shouldn't be here.